### PR TITLE
Removed extended-api-plugin in favour of  APIPlugin

### PR DESCRIPTION
**Pull request by jeffin143**
fix a bit of #2679 

- [x] Read and sign the [CLA][1]. PRs that haven't signed it won't be accepted.
- [x] Make sure your PR complies with the [writer's guide][2].
- [x] Review the diff carefully as sometimes this can reveal issues.
- [x] Do not abandon your Pull Request: [Stale Pull Requests][3].

### CODE REVIEW

**Comment by montogeek** [comment]
Could you please add links to both plugins?

**Comment by jeffin143** [comment]
> Could you please add links to both plugins?

Sorry didn't understand this command , since the file is deleted I don't think the link would redirect

File: src/content/migrate/5.md [hunk 1]
```diff
@@ -33,6 +33,8 @@ Some Plugins and Loaders might have a beta version that has to be used in order
 
 W> Check related Plugins and Loaders migration guide when upgrading across major versions.
 
+W>  ExtendedAPIPlugin is removed and the logic is merged into APIPlugin.
```
Comment (anikethsaha) [inline src/content/migrate/5.md]:
```suggestion
W>  ExtendedAPIPlugin is removed and the logic is merged into [`APIPlugin`](https://github.com/webpack/webpack/blob/master/lib/APIPlugin.js).
```

@jeffin143 I think @montogeek meant this .

Comment (jeffin143) [reply src/content/migrate/5.md]: Hey thanks :)

**Comment by EugeneHlushko** [comment]
Its fine, we don't need a link to removed plugin, its getting removed from the docs (still can be found in v4 docs no worries around that)

**Comment by EugeneHlushko** [comment]
Resolved a conflict here, lets see if lints fly

**Comment by EugeneHlushko** [comment]
Thanks!

**Pull request merged**